CHANGES.rst

```diff
@@ -131,6 +131,8 @@ barshadow (spec2 MOS)
   slit mode. Also correct the scale factors for error propagation when the
   correction is inverted. (`#9085
   <https://github.com/spacetelescope/jwst/issues/9085>`_)
+- Extrapolate barshadow corrections to cover wavelength gaps in reference
+  files. (`#9326 <https://github.com/spacetelescope/jwst/issues/9326>`_)
```

Comment:
Doesn't the fragment also need to be deleted? I don't see deletion in the original PR.

Reply:
> remove traces of them from the public release

Not sure if that is possible. You need a tag to do RC and all tags are public in a public repo. Anyways, that discussion is out of scope here anyway. Thanks for the clarification!